doc/crud.md

```diff
@@ -485,17 +713,34 @@ document:
     CDTDocumentRevision *updated = [datastore createDocumentFromRevision:rev
                                                                    error:&error];
     ```
+    ```swift
+    let rev = CDTDocumentRevision("doc2")
+    rev.body = saved.body
+    rev.body["hello"] = "world"
+    // Create att100 attachment
+    rev.attachments = saved.attachments
```

Comment:
Actually, I take that back. `Dictionary` in Swift is a `struct` which is a `Value` type. Which gives us some magic, see the [wwdc video](https://developer.apple.com/videos/play/wwdc2015-414/).